internal/tiger/cmd/auth_test.go

```diff
@@ -410,10 +409,6 @@ func mockOpenBrowser(t *testing.T) func(string) error {
 
 		// Give the OAuth server a moment to start
 		go func() {
-			// Sleep to ensure the OAuth callback server is listening
-			// This prevents "EOF" errors in CI when the server hasn't started yet
-			time.Sleep(100 * time.Millisecond)
```

Comment:
This was just a confused attempt to fix the issue previously, but isn't actually needed. Because we open the `net.Listener` synchronously [here](https://github.com/timescale/tiger-cli/blob/f0f657928abac4349de742045f1c4a621eac297f/internal/tiger/cmd/oauth.go#L124-L128), before we open the browser [here](https://github.com/timescale/tiger-cli/blob/f0f657928abac4349de742045f1c4a621eac297f/internal/tiger/cmd/oauth.go#L89-L91), it should never be the case that the server isn't ready to handle incoming connections by the time this function executes.